pydream/core.py

```diff
@@ -72,14 +84,22 @@ def run_dream(parameters, likelihood, nchains=5, niterations=50000, start=None,
         else:
 
             if type(start) is list:
-                args = zip([step_instance]*nchains, [niterations]*nchains, start, [verbose]*nchains, [nverbose]*nchains)
+                args = zip([step_instance]*nchains, [niterations]*nchains, start, [verbose]*nchains,
+                           [nverbose]*nchains, list(range(nchains)), chains_naccepts_iterations)
 
             else:
-                args = list(zip([step_instance]*nchains, [niterations]*nchains, [start]*nchains, [verbose]*nchains, [nverbose]*nchains))
+                args = list(zip([step_instance]*nchains, [niterations]*nchains, [start]*nchains, [verbose]*nchains,
```

Comment:
Was it intentional that args is a list here, but is a generator above in the `if` branch (no `list()` around the `zip()`)?
If not, you could do this in two steps with less duplication:
```python
if not isinstance(start, collections.abc.Iterable):
    start = [start] * nchains
args = list(zip([step_instance]*nchains, [niterations]*nchains, start, [verbose]*nchains, list(range(nchains)), chains_naccepts_iterations))
```